src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -1111,9 +1092,27 @@ SequenceDDLCommandsForTable(Oid relationId)
 						 "ALTER SEQUENCE %s OWNER TO %s", sequenceName,
 						 quote_identifier(ownerName));
 
+		/* commands to mark the object distributed in workers with metadata */
+		appendStringInfo(sequenceMarkDistStmt,
+						 "INSERT INTO citus.pg_dist_object (classid, objid, objsubid) "
+						 "(SELECT %u, c.oid, %d FROM pg_class c JOIN pg_namespace n ON (c.relnamespace = n.oid) "
+						 "WHERE c.relname = '%s' AND n.nspname = '%s') "
+						 "ON CONFLICT DO NOTHING;",
```

Comment:
We will likely prioritize syncing pg_dist_object. What do you think of leaving this out-of-scope for now? We may want to do things like using a UDF to allow regular users to perform this insert.